modules/eventing/partials/rbac-warning.adoc

```diff
@@ -0,0 +1,7 @@
+[CAUTION]
+====
+Changing the access role (i.e.,
+by revoking write permissions) could impact deployed eventing functions that have been assigned to this role. +
```

Comment:
Not important, but do you need the trailing `+`?

(Just asking 'cos I never understand where you have to use this... one of the weirder asciidoc features...)

Reply:
In this case, yes. 

I wanted the next line rendered separately, so I need the `+` to stop it appearing  on the same line.